code/crates/engine/src/consensus.rs

```diff
@@ -243,6 +243,7 @@ where
         input: ConsensusInput<Ctx>,
     ) -> Result<(), ConsensusError<Ctx>> {
         let height = state.height();
+        let value_payload = state.consensus.params.value_payload;
```

Comment:
Do we still use `value_payload` in core-consensus? If not, we should remove it from the consensus params.

Reply:
We still use it here:
- https://github.com/informalsystems/malachite/blob/anca%2Fengine_effect_handling/code/crates/core-consensus/src/handle/proposed_value.rs#L40
- https://github.com/informalsystems/malachite/blob/anca%2Fengine_effect_handling/code/crates/core-consensus/src/handle/proposal.rs#L78

Reply:
yes, we can look into changing that when we complete `ProposalOnly` implementation and maybe reorg based on latest discussions.

Reply:
For the second one I suggest we fix it in a different PR.

Reply:
moved signature verification in the engine, removed value_payload from the consensus parameters and put into the engine.